gradlew.bat

```diff
@@ -30,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=-Dfile.encoding=UTF-8 "-Xmx64m" "-Xms64m"
```

Comment:
Is the encoding utf8 provided by default now?

Reply:
Good question, I dunno actually 😂. I'd expect CI to surface any issues though as we're using the gradle wrapper there directly.